torch/csrc/distributed/rpc/BuiltinOp.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/Message.h>
+#include <torch/csrc/distributed/rpc/rpc_headers.h>
+#include <vector>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+// A buildin operator
+class TORCH_API BuiltinOp final {
+ public:
+  BuiltinOp(std::shared_ptr<Operator> op, std::vector<at::IValue>&& args);
+  ~BuiltinOp();
+
+  std::shared_ptr<Operator> op() const;
+  // return the argument stack of this builtin operator
+  std::vector<at::IValue>& stack();
```

Comment:
Returning a mutable reference here seems pretty weird to me. Do you really want to be mutating what is effectively your "opcode" for performing a mutable operation, when you actually run it? If so, it seems better to pass in the BuiltinOp as an rvalue reference to whatever runs it, to make it clear that the opcode gets "consumed" when you run it.